Makefile.PL

```diff
@@ -32,7 +32,21 @@ my %WriteMakefileArgs = (
   "VERSION" => "2.014",
   "test" => {
     "TESTS" => "t/*.t"
-  }
+  },
+  META_MERGE => {
+    'meta-spec' => { version => 2 },
+     resources => {
+         repository => {
+             type => 'git',
+             url  => 'https://github.com/rhandom/perl-net-server.git',
+             web  => 'https://github.com/rhandom/perl-net-server.git',
```

Comment:
What web url is wrong, it should be `https://github.com/rhandom/perl-net-server` instead.